app/utils/SplashLoadingScreen.ts

```
@@ -1,12 +1,16 @@
+import log from 'loglevel';
+
 const splashDiv = document.querySelector<HTMLElement>('div.splashPage');
 const activeClass = 'active';
 const fadeOutSpeed = 300;
 
 export default class SplashLoadingScreen {
     private static fadeOut() {
         if (!splashDiv) {
-            throw new Error('Splash page element not found');
+            log.error('Splash page element not found');
```

Why is this changed to an `log.error` instead of a `throw`? What's the reason we want to fail silently :smile: ? IMO if such an error happens in the app, we should not even continue as something is really broken and should be fixed by a developer

---

Failing tests unit tests in many places

---

Sooo... maybe we should get our unit tests setup to more closely resemble the actual UI, instead of changing the runtime logic to accommodate the tests? :slightly_smiling_face: I'm ok with creating a task to revert this change and add a `splashDiv` in the jsdom in unit tests :slightly_smiling_face:

---

Ok, ticket [RD-2841](https://cloudifysource.atlassian.net/browse/RD-2841).